recipes/beyla-golden-signals/README.md

```diff
@@ -121,6 +121,7 @@ kubectl annotate opentelemetrycollector otel \
     iam.gke.io/gcp-service-account=otel-collector@${GCLOUD_PROJECT}.iam.gserviceaccount.com
 ```
 
+this section is outdated
```

Comment:
@dashpole The sample doesn't produce `http_server_duration` since we are renaming it to `http_request_duration`. Two possible options:

1. The `transform` processor now has a `copy_metric` func which we could use to produce both isntead of renaming. Need to upgrade the collector
2. Update this README to tell the user to look at Applications dashboard instead

Wdyt?

Reply:
I would prefer not to copy the metric.  If you want, you can leave a commented line in that would switch to the other name.

Reply:
Which commented line? And just to clariy do you mean you want thsi sampel to only focus on the Applications dashboard? I could filter out all other [beyla metrics](https://grafana.com/docs/beyla/latest/metrics/) as well but I think http.client.duration is handy

Reply:
> Which commented line?

Maybe I misunderstood.  We just need one distribution at a time, right?

> And just to clariy do you mean you want thsi sampel to only focus on the Applications dashboard?

It depends on the resource usage difference.  If client metrics double the resource usage, it probably isn't worth it.  We can also mention this to the front end folks to see if they would build any experiences from client metrics if they were available.  If it is important enough that they would consider using them, we can include client metrics. I would start by excluding metrics that aren't used.

Reply:
Updated PTAL